shared/services/requirements.go

```diff
@@ -523,6 +526,11 @@ func waitBeaconClientSynced(c *cli.Context, verbose bool, timeout int64) (bool,
 	// Get BC status refresh time
 	bcRefreshTime := startTime
 
+	cfg, err := getConfig(c)
+	if err != nil {
+		return false, err
```

Comment:
ideally fmt.Errorf() wrap this error with %w and a little message about error getting config while checking bc